src/snowflake/snowpark/modin/plugin/extensions/general_overrides.py

```diff
@@ -101,513 +101,179 @@
 ]
 
 
+###########################################################################
+# Data manipulations
+###########################################################################
+
+
 @snowpark_pandas_telemetry_standalone_function_decorator
```

Comment:
can you update https://github.com/snowflakedb/snowpark-python/blob/3f605072e9204f9c67068f45668cbaf1960613b3/ci/check_standalone_function_snowpark_pandas_telemetry_decorator.py#L64 too?

Reply:
I think you meant Line 67, right? Done.